rom/laghos_rom.cpp

```diff
@@ -986,6 +988,16 @@ void ROM_Sampler::SetupEQP_Force(const CAROM::Matrix* snapX,
 
 void ROM_Sampler::Finalize(Array<int> &cutoff, ROM_Options& input)
 {
+	if (input.hyperreductionSamplingType == eqp_energy)
+	{
+		// For the energy-conserving EQP case, we increase the energy basis
+		// dimension by 1 to accomodate for the addition of the energy
+		// identity. 
+		// The change is done here, before the window basis parameters are
+		// written to their files, and is also carried to the caller's scope. 
+		cutoff[2] += 1;
+	}
```

Comment:
I increase the energy snapshots cutoff value by 1 here, so that the correct value is written to the parameter files (to be read in the online stage).